scripts/cmake/qskinny-macros.cmake

```diff
@@ -0,0 +1,13 @@
+if(TARGET Qt5::Svg)
+    ## @param SVG_FILENAME absolute filename to the svg
+    ## @param QVG_FILENAME absolute filename to the qvg
+    function(svg2qvg SVG_FILENAME QVG_FILENAME)
+        get_filename_component(SVG_FILENAME ${SVG_FILENAME} ABSOLUTE)
+        add_custom_command(
+            COMMAND svg2qvg ${SVG_FILENAME} ${QVG_FILENAME}
+            OUTPUT ${QVG_FILENAME}
+            DEPENDS ${SVG_FILENAME}
+            WORKING_DIRECTORY $<TARGET_FILE_DIR:Qt5::Svg>
+            COMMENT "Compiling ${SVG_FILENAME} to ${QVG_FILENAME}")
+    endfunction()
```

Comment:
@uwerat depending on how much "comfort" we want to provide we could also loop over all target's sources searching for *.svg but for now I think an explicit function call should be sufficient. 

Maybe the `add_custom_command` must become a `add_cusom_target`, CMake's dependency management for generated files is tricky